src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -38,6 +39,8 @@ public SyntaxTrivia ElasticMarker
         public SyntaxTrivia ElasticCarriageReturnLineFeed
             => SyntaxFactory.ElasticCarriageReturnLineFeed;
 
+        public ISyntaxKindsService SyntaxKinds { get; } = CSharpSyntaxKindsService.Instance;
```

Comment:
💡 Consider making `ISyntaxKindsService` exclusively provided through this service as `ISyntaxKinds`, and eliminating it as a standalone MEF export.

Reply:
I like having it both ways.  So i'm going to keep it hte same for now.